packages/table/src/common/utils.ts

```diff
@@ -408,3 +407,27 @@ export const Utils = {
         return approxCellHeight;
     },
 };
+
+/**
+ * Wrapper around document.getComputedStyle which applies some extra logic to exclude certain elements
+ * from the computation.
+ */
+function getComputedStyleWithExclusions(element: HTMLElement) {
+    const elementsToExclude = element.querySelectorAll(`.${CLASSNAME_EXCLUDED_FROM_TEXT_MEASUREMENT}`);
+
+    // hide all elements to exclude
+    for (let i = 0; i < elementsToExclude.length; i++) {
+        const el = elementsToExclude.item(i) as HTMLElement;
+        el.setAttribute("style", "display: none");
```

Comment:
we should save off the old `style` attribute value and set it back to what it was previously.